lib/ansible/modules/crypto/openssl_csr.py

```diff
@@ -261,6 +278,8 @@ def __init__(self, module):
         self.keyUsage_critical = module.params['keyUsage_critical']
         self.extendedKeyUsage = module.params['extendedKeyUsage']
         self.extendedKeyUsage_critical = module.params['extendedKeyUsage_critical']
+        self.basicConstraints = module.params['basicConstraints']
+        self.basicConstraints = module.params['basicConstraints_critical']
```

Comment:
same name as one line above, should be `self.basicConstraints_critical`

Reply:
Oops, fixed.